sqlmesh/dbt/test.py

```diff
@@ -133,6 +147,8 @@ def to_sqlmesh(self, context: DbtContext) -> Audit:
                     {source.sql_name for source in test_context.sources.values()}
                 ),
                 tags=self.tags,
+                hash_raw_query=True,
+                **self.sqlmesh_config_kwargs,
```

Comment:
Why do we pass these here?

Reply:
Is the question why here and not for audit? These are fields only the standalone audit has.

- `hash_raw_query` is used for the metadata hash.
- `sqlmesh_config_kwargs` are the configurable fields for sqlmesh `node_` concrete classes.

Reply:
Sorry, the diff UI confused me. I read this as if we were passing this into `add_globals(`